src/Microsoft.Android.Runtime.NativeAOT/Android.Runtime.NativeAOT/JavaInteropRuntime.cs

```diff
@@ -34,14 +42,15 @@ static void JNI_OnUnload (IntPtr vm, IntPtr reserved)
 	static void init (IntPtr jnienv, IntPtr klass)
 	{
 		try {
+			var log = GetDebugMonoLog ();
 			var typeManager = new NativeAotTypeManager ();
 			var options = new NativeAotRuntimeOptions {
 				EnvironmentPointer          = jnienv,
 				TypeManager                 = typeManager,
 				ValueManager                = new NativeAotValueManager (typeManager),
 				UseMarshalMemberBuilder     = false,
-				JniGlobalReferenceLogWriter = new LogcatTextWriter (AndroidLogLevel.Debug, "NativeAot:GREF"),
-				JniLocalReferenceLogWriter  = new LogcatTextWriter (AndroidLogLevel.Debug, "NativeAot:LREF"),
+				JniGlobalReferenceLogWriter = log.HasFlag (DebugMonoLog.Gref) ? new LogcatTextWriter (AndroidLogLevel.Debug, "NativeAot:GREF") : null,
+				JniLocalReferenceLogWriter  = log.HasFlag (DebugMonoLog.Lref) ? new LogcatTextWriter (AndroidLogLevel.Debug, "NativeAot:LREF") : null,
```

Comment:
I basically commented these lines to test performance, but I notice there are also some messages like:
```
06-20 04:07:15.565  8457  8457 I NativeAotTypeManager: # jonp: GetTypesForSimpleReference: jniSimpleReference=`android/view/DisplayListCanvas`
06-20 04:07:15.565  8457  8457 I NativeAotTypeManager: # jonp: GetTypesForSimpleReference: jniSimpleReference=`android/graphics/Canvas`
06-20 04:07:15.565  8457  8457 I NativeAotFromAndroid: # jonp:   GetTypesForSimpleReference: jniSimpleReference=`android/graphics/Canvas` -> `Android.Graphics.Canvas`
```
Should we remove these, or use the same flag? (this can be a different PR)